src/matlab2tikz.m

```diff
@@ -1473,14 +1474,21 @@ case guitypes()
             % axis styles (e.g., colors).
             options = opts_add(options, 'separate axis lines');
         end
```

Comment:
I am unsure, if the chunck below is needed or correct. 

a) Is there a possibility to change the color in matlab
b) if there is such a possibility, why always use the same color
c) As far as i know setting YColor and friends worked fine.